### ISSUE TITLE
riak_kv_eleveldb_backend fails intermittently

### ISSUE THREAD

**Issue by reiddraper** [basho/riak_kv#928]
I've only been able to reproduce this with a `make test`, and on a [builder doing that same](http://buildbot.bos1/builders/test-riak_kv/builds/1315/steps/shell_3/logs/stdio). The error is:

```
test/backend_eqc.erl:132:<0.9658.2>: Result: {exception,
            {'EXIT',
                {badarg,
                    [{eleveldb,close,[<<>>],[]},
                     {riak_kv_eleveldb_backend,stop,1,
                         [{file,"src/riak_kv_eleveldb_backend.erl"},
                          {line,151}]},
                     {eqc_statem,run_commands,2,
                         [{file,"../src/eqc_statem.erl"},{line,737}]},
                     {eqc_fsm,run_commands,2,
                         [{file,"../src/eqc_fsm.erl"},{line,658}]},
                     {backend_eqc,'-prop_backend/4-fun-2-',2,
                         [{file,"test/backend_eqc.erl"},{line,124}]}]}}}
```

And the full output (including shrunk test) is in [the logs](http://buildbot.bos1/builders/test-riak_kv/builds/1315/steps/shell_3/logs/stdio). I've run the test by itself for 30 minutes, with no sign of the bug. I'm guessing it's somehow state-related from another test.

```
erl -pa .eunit deps/*/ebin
P = backend_eqc:prop_backend(riak_kv_eleveldb_backend, false, [{data_root, "test/eleveldb-backend"}], fun(BeState,_Olds) -> catch(riak_kv_eleveldb_backend:stop(BeState)) end).
eqc:quickcheck(eqc:testing_time(60 * 30, P)).
```

**Comment by reiddraper** [comment]
This is a similar failure to basho/eleveldb#61.

**Comment by reiddraper** [comment]
Fixed in basho/eleveldb#113.
